scripts/index-client.js

```diff
@@ -1,23 +1,35 @@
-// md + assets
+// assets
 require.context("icons", true, /\.svg$/)
-require.context("../content", true, /\.(md|html|ico|jpe?g|png|gif)$/)
+require.context("../content", true, /\.(html|ico|jpe?g|png|gif)$/)
 
 // OMG GLOBAL STYLES !
 import "_legacy-css/index.css"
 
 // statinamic
-import "babel-core/polyfill"
+import "babel-polyfill"
 import "whatwg-fetch"
-
 import statinamicClient from "statinamic/lib/client"
 
+import * as layouts from "layouts"
+import metadata from "app/metadata"
 import routes from "app/routes"
 import store from "app/store"
 
-import metadata from "./metadata"
-
 statinamicClient({
+  layouts,
   metadata,
   routes,
   store,
 })
+
+// md files → JSON && generate collection + hot loading for dev
+let mdContext = require.context("../content", true, /\.md$/)
+mdContext.keys().forEach(mdContext)
```

Comment:
try removing the line, and the all things just don't work. require.context is so weird.